### PR TITLE
Like context

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added like/remove  like  to a video in app
fixed : route for remove like video

### CODE REVIEW

File: src/Context/LikeContext.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { createContext, useState, useContext, useEffect } from 'react';
+import axios from 'axios'
+import { useVideo } from './VideoContext';
+import { useNavigate } from 'react-router-dom';
+// import { Navigate } from 'react-router-dom';
```
Comment (zoyatanweer) [inline src/Context/LikeContext.js]: remove if not required

File: src/Context/VideoContext.js [hunk 2]
```diff
@@ -10,14 +10,21 @@ const VideoProvider = ({children}) => {
   const [ playerArr, setPlayerArr] = useState([])
 
   const playerHandler = (_id) => {
-    const obj = videos.find((video) => video._id === _id);
+    const obj = videoList.find((video) => video._id === _id);
     setPlayerArr([obj])
   }
 
-  console.log(playerArr)
```
Comment (zoyatanweer) [inline src/Context/VideoContext.js:17]: remove console.log

**Review by zoyatanweer — APPROVED** [review]
- code looks clean and good
- great UI
- good to merge

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by Shubhamkr2610 — APPROVED** [review]
everything looks good to me. Nothing suggestion from my side. You can proceed for merge

**Pull request merged**